### ISSUE TITLE
px-to-rem.number-of-decimals-digits set problem

### ISSUE THREAD

**Issue by evanyangg** [sainoba/vscode-px-to-rem#7]
px to rem(75)
eg: 26 / 75 ≈ 0.35rem
then rem to px 
eg: 0.35rem * 75 = 26.25px
i want get 26, so i set "px-to-rem.number-of-decimals-digits" 0 is not working

```
function px2Rem(px, pxPerRem) {
    if (pxPerRem == 0) { return 0; }
    const config = vscode.workspace.getConfiguration("px-to-rem");
    var maxDecimals = config.get('number-of-decimals-digits');
    maxDecimals = **Math.max(1, maxDecimals);**
    const value = parseFloat((px / pxPerRem).toFixed(maxDecimals));
    return value;
}
```

**Comment by sainoba** [comment]
Hello evanyangg, thank you very much for taking the time to find the issue.
I'll try to fix it asap. :+1:

**Comment by sainoba** [comment]
Hi it should be fixed in the newest version 1.2.7 :slightly_smiling_face:

**Comment by evanyangg** [comment]
I have seen it, it is very efficient, thank you very much.
